### ISSUE TITLE
Failed assertion: boolean expression must not be null

### ISSUE THREAD

**Issue by trevorwang** [Clancey/simple_auth#71]
`useNonce` was not initialized

```
^
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: Failed assertion: boolean expression must not be null
#0      WebAuthenticator.getInitialUrlQueryParameters 
package:simple_auth/…/api/webAuthenticator.dart:56
<asynchronous suspension>
#1      AzureADAuthenticator.getInitialUrlQueryParameters 
package:simple_auth/…/providers/azureAD.dart:69
<asynchronous suspension>
#2      WebAuthenticator.getInitialUrl 
package:simple_auth/…/api/webAuthenticator.dart:45
<asynchronous suspension>
#3      SimpleAuthFlutter.showAuthenticator 
package:simple_auth_flutter/simple_auth_flutter.dart:29
<asynchronous suspension>
#4      OAuthApi.performAuthenticate 
package:simple_auth/…/oauth/oauthApi.dart:86
<asynchronous suspension>
#5      AuthenticatedApi.authenticate 
package:simple_auth/…/api/authenticatedApi.dart:20
<asynchronous suspension>
#6      Azure.login 
package:flutter_zhihu/azure.dart:32
<asynchronous suspension>
#7      _azureSign.<anonymous closure> (package:flut<…>
flutter: 2019-03-01 13:17:16.951067: LoginPage: FINE: SignStatus.SigningIn
```

**Comment by damccoy1** [comment]
Fixed in #70 

You can use the following dependency for your `pubspec.yaml` to use the latest code from GitHub instead of the published version:
```
simple_auth_flutter: 
    git:
      url: https://github.com/Clancey/simple_auth.git
      path: simple_auth_flutter
```

**Comment by Clancey** [comment]
Already published
